scripts/merge-measures-data.js

```diff
@@ -47,9 +47,70 @@ function mergeQpp(qppJson) {
       if (performanceRateInfo.metricType) {
         qppJson[index].metricType = performanceRateInfo.metricType;
       }
+
     }
   });
 
   console.error('did not find measure details for the following', measuresNotFound);
+
+  // separate passes for ecqm data to keep things simple
+  // load extracted ecqm data
+  const generatedEcqmStrataJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../util/generated-ecqm-data.json'), 'utf8'));
+
+  qppJson.forEach(function(qppItem, index) {
+    if (qppItem.category !== 'quality') return;
+    const ecqmInfo = _.find(generatedEcqmStrataJson, {'eMeasureId': qppItem.eMeasureId});
+    if (!ecqmInfo) return;
+
+    qppJson[index].eMeasureUuid = ecqmInfo.eMeasureUuid;
+    qppJson[index].metricType = ecqmInfo.metricType;
+    const oldStrata = qppJson[index].strata;
+
+    // if none, just set it
+    if (oldStrata.length === 0) {
+      qppJson[index].strata = ecqmInfo.strata;
+    } else { // only override description, uuids
+      ecqmInfo.strata.forEach((newStratum, ecqmIndex) => {
+        if (oldStrata[ecqmIndex]) {
+          qppJson[index].strata[ecqmIndex].eMeasureUuids = newStratum.eMeasureUuids;
+          qppJson[index].strata[ecqmIndex].description = newStratum.description;
+        } else {
+          qppJson[index].strata[ecqmIndex] = newStratum;
+        }
+      });
+      if (oldStrata.length > 1) {
+        // ASSUMPTION: strata already available are in the same order as the ones extracted from the xml files
```

Comment:
The ordering is derived from the spec, so as long as we are generally always pulling things in the correct order from the spec we should be okay. Still, yeah, kinda 😬